katpoint/timestamp.py

```diff
@@ -95,11 +95,11 @@ def __str__(self):
 
     def __eq__(self, other):
         """Test for equality"""
-        return self.secs == other.secs
+        return float(self) == float(other)
```

Comment:
I like this trick! Since `self` is guaranteed to be a `Timestamp`, I would go with:
```Python
return self.secs == float(other)
```